src/Sylius/Behat/Context/Ui/Admin/ManagingPaymentMethodsContext.php

```diff
@@ -460,15 +496,15 @@ public function iConfigureItForUsernameWithSignature($username, $signature)
     /**
      * @When I do not specify configuration password
      */
-    public function iDoNotSpecifyConfigurationPassword()
+    public function iDoNotSpecifyConfigurationPassword(): void
     {
         // Intentionally left blank to fulfill context expectation
     }
 
     /**
      * @When I configure it with test stripe gateway data
      */
```

Comment:
```suggestion
    /**
     * @When I configure it with test stripe gateway data
     *
     * @throws InvalidArgumentException
     */
```

Reply:
I am unsure if this method is throwing an exception.